### DISCUSSION TITLE
Loading issue templates/forms from another repository?

### DISCUSSION THREAD

**Discussion by dotansimha** [community/community#35155] [Projects and Issues]
Hi,

I was wondering if it's possible to load issues/PRs templates/forms from another repository? like a reference to a remote repository that contains the actual file? 
We are a group of developers that maintains many repositories together, and we wish to centralize our configurations (we did it for GitHub Actions with great success!)

**Comment by MAN-Sendance** [comment]
Really good question, I am also interested in this topic.
